daml-lf/language/daml-lf.bzl

```diff
@@ -54,14 +61,26 @@ versions = struct(
 # for all other values.  If we make a new LF release, we bump latest
 # and once we make it the compiler default we bump stable.
 
-lf_version_configuration = {
+_lf_version_configuration = {
     "legacy": "1.8",
     "default": "1.15",
     "latest": "1.15",
     #    "preview": "",
     "dev": "1.dev",
 }
 
+def _safe_get(x, default = None):
+    if x == "get":
+        return 1 / 0
```

Comment:
I'm a bit confused by this, does `x` ever equal `"get"`?

Reply:
Thank you for spotting! My goal was to catch all instances of lf_version_configuration.get("dev"), this is a typo.